.circleci/config.yml

```
@@ -352,13 +352,13 @@ jobs:
     resource_class: 'medium+'
     steps:
       - checkout
+      - install_js
       - run:
           name: Resolve typescript version
           command: |
             pnpm add typescript@next -d -w
```

You could try with the following command. (Not sure it will work)
```suggestion
            pnpm update -r typescript@next
```

---

It "works": https://app.circleci.com/pipelines/github/mui/material-ui/136906/workflows/efb61b97-5715-455c-915f-a008c66cd81d/jobs/738078. It fails silently because this job is meant to fail silently.

The previous command also resulted in the same error: https://app.circleci.com/pipelines/github/mui/material-ui/136902/workflows/b9c1cbb1-625e-4e3d-a68b-4da986bd0108/jobs/738050?invite=true#step-107-198

Looks like the `docs:typescript` command is the one failing, but probably something to look into another PR.